client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -143,7 +151,7 @@
                 $scope.origDictionary = result;
                 $scope.dictionary = _.create(result);
                 $scope.dictionary.content = _.create(result.content || {});
-                $scope.dictionary.is_active = $scope.dictionary.is_active !== false;
+                $scope.dictionary.is_active = $scope.dictionary.is_active !== 'false';
```

Comment:
Oh, OK then, I guess we'll have to live with it then. 

As it stands, there's probably no good way of transparently doing the `bool` to `str` (and vice versa) conversions? E.g. when data enters/leaves the application boundary?